<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1331</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EFEFF2;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="type_test_choose">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_questions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>321</width>
       <height>591</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>961</width>
       <height>751</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="test_question">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>951</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
padding-left:10px</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите вопрос</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>86</y>
        <width>951</width>
        <height>661</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTextEdit" name="answer_1">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
padding-left:10px</string>
           </property>
           <property name="placeholderText">
            <string>Введите ответ и выберите верные</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="choose_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QTextEdit" name="answer_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
padding-left:10px</string>
           </property>
           <property name="placeholderText">
            <string>Введите ответ и выберите верные</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="choose_2">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>200</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTextEdit" name="answer_3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
padding-left:10px</string>
           </property>
           <property name="placeholderText">
            <string>Введите ответ и выберите верные</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="choose_3">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTextEdit" name="answer_4">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
padding-left:10px</string>
           </property>
           <property name="placeholderText">
            <string>Введите ответ и выберите верные</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="choose_4">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="add_new_question">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>680</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #0171CA;
color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>новый вопрос</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_curent_question">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>680</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #0171CA;
color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>удалить вопрос</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_curent_question">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>730</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #0171CA;
color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>сохранить вопрос</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
